examples/lighting-app/nrfconnect/CMakeLists.txt

```
@@ -20,6 +20,8 @@ get_filename_component(CHIP_APP_SERVER ${CHIP_ROOT}/examples/common/chip-app-ser
 get_filename_component(NRFCONNECT_COMMON ${CHIP_ROOT}/examples/platform/nrfconnect REALPATH)
 get_filename_component(LIGHTING_COMMON ${CHIP_ROOT}/examples/lighting-app/lighting-common REALPATH)
 
+list(PREPEND OVERLAY_CONFIG ${CHIP_ROOT}/config/nrfconnect/overlay-sample-defaults.conf)
```

I'm not sure if this is a problem in the CHIP, but NCS itself uses CMake 3.13 where `PREPEND` is not supported. I guess you can use `INSERT` method on 0 index. Same for other places in this PR.

```suggestion
list(INSERT OVERLAY_CONFIG 0 ${CHIP_ROOT}/config/nrfconnect/overlay-sample-defaults.conf)
```

---

Done